src/components/GameListItem.vue

```diff
@@ -16,17 +16,28 @@
         <p>{{ readyText }} players</p>
       </v-col>
       <v-col cols="3" class="list-item__button">
+        <!-- Join Button -->
         <v-btn
-          color="primary"
-          rounded
-          variant="outlined"
-          min-width="200"
+          v-if="!isSpectatable"
+          v-bind="buttonAttrs"
```

Comment:
I know why you moved these since you're re-using them, but it makes it harder to figure out the button style at a glance. Don't hate it, but there's a slight cognitive impact now.

Reply:
Yeah the alternative as I see it is to make it just one button with computed props for the text, icon, data-cy attribute, and a higher order click handler that delegates to the correct method based on the prop for spectating. My hunch is that this is simpler; penny for your thoughts